test/functional/terminal/scrollback_spec.lua

```diff
@@ -441,8 +439,7 @@ describe("'scrollback' option", function()
     -- 'scrollback' option is synchronized with the internal sb_buffer.
     command('sleep 100m')
     if iswin() then
-      nvim('command',
-      'call jobsend(b:terminal_job_id, "for($i=1;$i -le 40;$i++){Write-Host \\"line$i\\"}\\<CR>")')
+      feed_data('for($i=1;$i -le 40;$i++){Write-Host \"line$i\"}\13') -- \13 = <CR>
```

Comment:
Is there a reason `\n` can't be used instead of `\13`? It would be more consistent, since the non-windows case does that.